.github/workflows/frontend_pr_checks.yml

```diff
@@ -0,0 +1,67 @@
+name: Frontend PR Checks
+
+on:
```

Comment:
To match when the backend tests run.

```diff
- on:
-   pull_request:
-     branches:
-         - main

+ on:
+  push:
+    branches:
+      - main
+  pull_request:
```

Reply:
do you think I should still keep the `paths`? just a matter of how often we want to run these tests, I suppose

Reply:
I think it depends. I haven't looked close enough at the test suite for the front end yet. Does it make calls to the back end in tests, or is it all mocked out? If it makes the actual calls it would be good to make sure changes to the backend don't break the front end. If they are mocked the paths make sense.

Reply:
hm yeah right now they only call frontend stuff, but eventually perhaps we will add cypress for e2e tests. I'll put `paths` in for now to save on the CI minutes for now, but yeah if we do have e2e later we should change it